contrib/pkg/createcluster/create.go

```diff
@@ -196,6 +202,11 @@ func (o *Options) Validate(cmd *cobra.Command) error {
 	if !o.UseClusterImageSet && len(o.ClusterImageSet) > 0 {
 		cmd.Usage()
 		log.Info("If not using cluster image sets, do not specify the name of one")
+		return fmt.Errorf("invalid option")
+	}
+	if len(o.ServingCert) > 0 && len(o.ServingCertKey) == 0 {
```

Comment:
Should this be the following instead to also validate that the key was not provided with the cert?
```
if (len(o.ServingCert) == 0) != (len(o.ServingCertKey) == 0) {
```

Reply:
What triggers adding a cert section is the cert param, if you only specify a key, it's just ignored. (Maybe shouldn't be?)